lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -1845,3 +1845,10 @@ galaxy:
   # reason to disable it.
   #use_pbkdf2: true
 
+  # Tell Galaxy that multiple domains sharing the same root are associated
+  # to this instance and wants to share the same session cookie.
+  # This allow a user to stay logged in when passing from one subdomain
+  # to the other.
+  # This root domain will be written in the unique session cookie shared
+  # by all subdomains.
+  #multidomains_root: null
```

Comment:
maybe name it something like `cookie_domain_root`? or something to make it more clear related to the cookies?

Reply:
Or just `cookie_domain`, which is also the name of property in the cookie ?

Reply:
+1!

Reply:
Yeah, that's perfect.  Once you swap that and rebuild the config this is good to go IMO; nice improvement here.